module/zfs/dsl_scan.c

```diff
@@ -147,7 +147,7 @@ unsigned long zfs_scan_vdev_limit = 4 << 20;
 
 int zfs_scan_issue_strategy = 0;
 int zfs_scan_legacy = B_FALSE; /* don't queue & sort zios, go direct */
-uint64_t zfs_scan_max_ext_gap = 2 << 20; /* in bytes */
+unsigned long zfs_scan_max_ext_gap = 2 << 20; /* in bytes */
```

Comment:
I forgot i had this change in another branch, it seems it could be squeezed in this PR.

@tcaputi  hope this is not an issue (`uint64_t` -> `unsigned long`), otherwise i don't know how to expose this as a tunable (original code is from d4a72f2).

Reply:
looking at the code that should be ok.